microcosm_flask/fields/query_string_list.py

```diff
@@ -22,7 +27,7 @@ def _deserialize(self, value, attr, obj):
 
         try:
             attribute_elements = [attr_element.split(",") for attr_element in obj.getlist(attr)]
-            attribute_params = [param for attr_param in attribute_elements for param in attr_param]
+            attribute_params = SelfSerializableList(param for attr_param in attribute_elements for param in attr_param)
```

Comment:
1. done
2. this is deserialization so we want to return a list because we want to have list as internal data type, this deserialization is used for changing query string parameter into internal attribute and we want this list to be able to print nicely back via `str`